### ISSUE TITLE
ccross -a show empty string if ccross-curl not in PATH

### ISSUE THREAD

**Issue by ampolyak** [MasterSoft24/CloudCross#44]
**ccross -a** show empty string instead of confirm application credentials URL if **ccross-curl** not in **PATH**

**Comment by MasterSoft24** [comment]
Yes it is. Both binaries **ccross** **and ccross-curl** must be in /usr/bin or in other system-wide binary folder. For a test reason you could add to PATH your path with these binaries

**Comment by ampolyak** [comment]
but maybe it will be better to print something like "ccross-curl not found" instead of providing empty line?

**Comment by MasterSoft24** [comment]
hmm, I think it can be done ))

**Comment by ampolyak** [comment]
:thumbsup:
